packages/ag-charts-enterprise/src/series/sunburst/sunburstSeries.ts

```diff
@@ -607,11 +608,24 @@ export class SunburstSeries extends _ModuleSupport.HierarchySeries<_ModuleSuppor
         return [];
     }
 
-    override getSeriesDomain() {
-        return [NaN, NaN];
-    }
-
-    override getLegendData() {
-        return [];
+    override animateEmptyUpdateReady({ datumSelections }: any) {
+        fromToMotion<_Scene.Group, any, _ModuleSupport.HierarchyNode>(
```

Comment:
Can we get rid of the `any`s here? The idea is that the `toFn()` / `fromFn()` should by typesafe if the typings are correct?